src/main/java/frc/robot/RobotContainer.java

```diff
@@ -110,36 +112,71 @@ public RobotContainer(RobotBase robot) {
    */
   
   private void configureBindings() {
-      Joystick leftJoystick = new Joystick(0);
-      Joystick rightJoystick = new Joystick(1);
-      Joystick altJoystick = new Joystick(2);
-
-
-      swerveDrive.setDefaultCommand(new RunCommand(() -> {
-        if (robot.isTeleopEnabled()){
-          //drive
-          swerveDrive.drive(
-            limitX.calculate(applyDeadband(-leftJoystick.getX(), DrivetrainConstants.DRIFT_DEADBAND))*DriverConstants.speedMultiplier,
-            limitY.calculate(applyDeadband(-leftJoystick.getY(), DrivetrainConstants.DRIFT_DEADBAND))*DriverConstants.speedMultiplier,
-            applyDeadband(-rightJoystick.getX(), DrivetrainConstants.ROTATION_DEADBAND)*DriverConstants.angleMultiplier);
-        }
-        else 
-        {
-          swerveDrive.drive(0,0,0);
-        }
-        
-      }, swerveDrive));
+    Joystick leftJoystick = new Joystick(0);
+    Joystick rightJoystick = new Joystick(1);
+    Joystick altJoystick = new Joystick(2);
+
+    frontLeftWheel.getEncoder().setInverted(true);
```

Comment:
Small thing, but I'd move this to a different function for better organization, and just leave this function for button bindings. See last year's [configureObjects](https://github.com/frc1418/2023-robot/blob/7d363f890eacad68db73f5562156c8f05406879e/src/main/java/frc/robot/RobotContainer.java#L207) function for reference

Reply:
👍